v2/components/schemas/Snap.yaml

```diff
@@ -0,0 +1,96 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: Detailed information about a single snap.
+properties:
+  id:
+    type: string
+    description: The unique identifier for the snap.
+  name:
+    type: string
+    description: The name of the snap.
+  base:
+    type: string
+    description: The base snap this snap is built on.
+    example: core20
+  channel:
+    type: string
+    description: The default channel for this snap.
+  common-ids:
+    type: array
+    items:
+      type: string
+    example: code.desktop
+  confinement:
+    type: string
+    enum: 
+      - strict
+      - classic
+  contact:
+    type: string
+    description: The primary contact URL or email for the developer.
+  description:
+    type: string
+    description: A detailed description of the snap.
+  developer:
+    type: string
+    description: The username of the snap developer.
+  devmode:
+    type: boolean
+    description: Whether the snap can be installed in devmode.
+  download-size:
+    type: integer
+    format: int64
+    description: The size of the snap package in bytes.
+  icon:
+    type: string
+    format: uri
+    description: A URL to the snap's icon image.
+  license:
+    type: string
+  private:
+    type: boolean
+    description: Whether the snap is private.
+  revision:
+    type: string
+    description: The latest revision number of the snap in this channel.
+  status:
+    type: string
+    example: available
+  store-url:
+    type: string
+    format: uri
+    description: The URL to the snap's page in the Snap Store.
+  summary:
+    type: string
+    description: A short, one-line summary of the snap.
+  title:
+    type: string
+  type:
+    type: string
+    enum: 
+      - app
+      - kernel
+      - os
```

Comment:
Can you document that "os" is deprecated and was only used for the "core" and "ubuntu-core" snaps, historically.